group-coordinator/src/main/java/org/apache/kafka/coordinator/group/Group.java

```diff
@@ -200,4 +200,13 @@ void validateOffsetFetch(
      * Requests a metadata refresh.
      */
     void requestMetadataRefresh();
+
+    /**
+     * Returns whether this group should be expired or not
+     *
+     * @return whether the group should be expired
```

Comment:
nit: all the java doc in this file ends with period (`.`). Could you add it? Thanks.